src/app/components/dashboard/performance-item.component.html

```diff
@@ -0,0 +1,88 @@
+<!-- 
+    Copyright (C) 2020 - 2023 Bitfly GmbH
+    
+    This file is part of Beaconchain Dashboard.
+
+    Beaconchain Dashboard is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Beaconchain Dashboard is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Beaconchain Dashboard.  If not, see <http://www.gnu.org/licenses/>.
+ -->
+
+<div class="grid">
+	<div class="grid-cell left" (click)="unit.switchCurrencyPipe()">
+		<div class="grid">
+			<ion-label class="value left" [class]="performanceData.performance1d | valuestyle : 0">
+				{{ performanceData.performance1d | mcurrency : currency : unit.pref.Cons }}
```

Comment:
**Question:** This (and some other values) has been `unit.pref.Exec` before for the execution performance, see https://github.com/gobitfly/eth2-beaconchain-explorer-app/pull/262/files#diff-176bd5938e1566da4de6f22f275b3298a13f42789fea88d54bc04b5729085df5L374

Is this change intentional?